islandora_basic_collection.install

```diff
@@ -43,7 +43,7 @@ function islandora_basic_collection_uninstall() {
  * Default collection paging update.
  */
 function islandora_basic_collection_update_7001() {
-  $old_var = variable_get('islandora_basic_collection_page_size');
+  $old_var = variable_get('islandora_basic_collection_page_size', 12);
```

Comment:
@whikloj so why would we like to change people's saved value? I mean, if they went to that config page, saw 10 there and saved, that 10 means they saved it. If they never saved it, then that 10 does not exist. I'm pretty sure there is an explanation i'm missing